glusterd2/xlator/options/options.go

```diff
@@ -254,21 +260,37 @@ func ValidatePriorityOrSize(o *Option, val string) error {
 
 // ValidateSizeList validates if the option is a valid size list
 func ValidateSizeList(o *Option, val string) error {
-	return ErrInvalidArg
+	var sizeinbytes int64
+	slist := strings.Split(val, ",")
+	for _, el := range slist {
+		l := len(el)
```

Comment:
You don't need this additional variable here. You can simple use `len(el)` below.

```go
size := el[len(el)-2:]
```

Reply:
Hi @prashanthpai am using 'l' twice, I don't think it'll be a good idea to compute length twice.
Using l to compute 'size', the size type(KB, MB) and 't', trimmed string.